packages/api/src/parse-cli.ts

```diff
@@ -78,14 +80,19 @@ export type CliArgs = ReturnType<typeof parseCli>;
 // before passing it to yargs completely breaks type inference, which is a huge shame.
 // So... this monkeypatches yargs to capture that variable. If you know of a more
 // elegant way, I'd love to hear it!
-let args;
+let allOptions;
 const originalOpts = yargs.options;
-yargs.options = function (arg) {
-  args = arg;
-  return originalOpts.call(this, arg);
+yargs.options = function (...args) {
+  if (!allOptions) {
+    allOptions = args[0];
```

Comment:
Hm, I think we sent all process args to that `yargsToMist` function. Won't this break that, not being able to display the help from `-j` arg?

If `args` is actually a list of lists, mind adding a type to the `allOptions` above making it clear what it is?

Reply:
Yeah, I need to add a type definition to this stuff to unbreak CI. Literally all I'm trying to do here is get the first `.options` call that we're invoking here; all the subsequent calls are apparently internal https://github.com/livepeer/studio/blob/357a24e9a39f5c441cc89270a4415cc857138c93/packages/api/src/parse-cli.ts#L97